### PR TITLE
Update project structure

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

**Comment by henrykironde** [comment]
@apoorvaeternity feel free to review these changes

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  apt:
+    sources:
+      - travis-ci/sqlite3
+    packages:
+      - sqlite3
+services:
+  - postgresql
+
+install:
+  - pip install retriever
```
Comment (apoorvaeternity) [inline .travis.yml]: Let's install using requirements.txt 
`pip install -r retrieverdash/requirements.txt`

File: retrieverdash/README.rst [hunk 2]
```diff
@@ -1,46 +1,36 @@
-django-project-skeleton
-=======================
+Retriever Status Dashboard
+==========================
+
+**Retriever Status Dashboard** provides details and status info....
```
Comment (apoorvaeternity) [inline retrieverdash/README.rst]: **Retriever Status Dashboard** provides details and status info of datasets available through **retriever**.

**Review by apoorvaeternity — CHANGES_REQUESTED** [review]
Overall it looks good to me. Just a few suggestions.

File: retrieverdash/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 Django==2.0.2
 pytz==2017.3
+git@github.com:weecology/retriever.git
```
Comment (apoorvaeternity) [inline retrieverdash/requirements.txt]: Is this working? I don't think that should work. I think `git+https://www.github.com/weecology/retriever@master` would be better.

**Review by apoorvaeternity — CHANGES_REQUESTED** [review]

File: retrieverdash/requirements.txt (hunk 3 above)
Comment (henrykironde) [reply retrieverdash/requirements.txt]: correct

**Review by apoorvaeternity — APPROVED** [review]

**Pull request merged**